src/signals/incident/components/IncidentForm/services/custom-validators/index.js

```diff
@@ -0,0 +1,17 @@
+export function validateFileType(file, meta) {
+  if (meta && meta.allowedFileTypes && Array.isArray(meta.allowedFileTypes)) {
+    if (meta.allowedFileTypes.indexOf(file.type) === -1) {
+      return { custom: 'Dit bestand heeft niet het juiste type.' };
```

Comment:
It might be nice to say something like 'Dit bestand heeft type <type> en wordt niet ondersteund. Ondersteund worden: [..., ..., ...].'